packaging/installer/install.ps1

```diff
@@ -427,12 +470,47 @@ if (!(check_if_admin)) {
 echo 'Checking execution policy'
 check_policy
 
-if (msi_installed -name "Splunk OpenTelemetry Collector") {
-    throw "The Splunk OpenTelemetry Collector is already installed. Remove or uninstall the Collector and rerun this script."
-}
-
 if (Get-Service -Name $service_name -ErrorAction SilentlyContinue) {
-    throw "The $service_name service is already installed. Remove or uninstall the Collector and rerun this script."
+    Write-Host "The $service_name service is already installed. Checking installation for automatic update."
+
+    $uninstall_collector = $true
+    $collector_sids = get_msi_installation_sids -product_name $CollectorServiceDisplayName
+    if ($collector_sids.Count -eq 0) {
+        $uninstall_collector = $false
+        Write-Warning "The $service_name service exists but it is not on the Windows installation database."
+    }
+    else {
+        if ($collector_sids.Count -gt 1) {
+            $sids_list = $collector_sids -join ", "
+            throw "The Splunk OpenTelemetry Collector is already installed for multiple users (SIDs: $sids_list). Uninstalling the collector and remove remaining users installations."
```

Comment:
The message here makes it seem like the script is proceeding with the uninstallation process, specifically `Uninstalling the collector...` Is this meant to be telling the user to do this, or something else?